cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -37,14 +20,33 @@
 import no.unit.nva.publication.s3imports.FileContentsEvent;
 import nva.commons.core.JsonUtils;
 
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static no.unit.nva.cristin.mapper.CristinObject.MAIN_CATEGORY_FIELD;
+import static no.unit.nva.cristin.mapper.CristinObject.PUBLICATION_OWNER_FIELD;
+import static no.unit.nva.cristin.mapper.CristinObject.SECONDARY_CATEGORY_FIELD;
+import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValues;
+import static no.unit.nva.hamcrest.DoesNotHaveEmptyValues.doesNotHaveEmptyValuesIgnoringFields;
+import static nva.commons.core.attempt.Try.attempt;
+import static org.hamcrest.MatcherAssert.assertThat;
+
 public class CristinDataGenerator {
 
     public static final int SMALL_NUMBER = 10;
     public static final Random RANDOM = new Random(System.currentTimeMillis());
     public static final Faker FAKER = Faker.instance();
     public static final int FIRST_TITLE = 0;
     public static final ObjectMapper OBJECT_MAPPER = JsonUtils.objectMapperNoEmpty.configure(
-        SerializationFeature.INDENT_OUTPUT, false);
+            SerializationFeature.INDENT_OUTPUT, false);
```

Comment:
in a new PR use the new commons and the obejctmapper that serializes everything in one line.